docs/getting-started.md

```diff
@@ -1,6 +1,6 @@
 Getting XQEMU
 -------------
-**Download for Windows:** The latest, pre-built release version of XQEMU for Windows can be [**downloaded here**](https://github.com/xqemu/xqemu/suites/373368695/artifacts/830137).
+**Download for Windows:** If you have a GitHub account, the latest pre-built release version of XQEMU for Windows can be [**downloaded here**](https://github.com/xqemu/xqemu/suites/373368695/artifacts/830137).
```

Comment:
I'm against this change because we should not encourage users to create a GitHub account.

I wanted to manually update these links to a working URL (likely a file attachment on a GitHub issue) but all my other xqemu.com PRs never got reviewed (and they move quite a lot of information so I was worried about merge conflicts).

Regardless of my plans, this should be a separate PR.

Reply:
Leaving it as-is just leads to user confusion. It either needs to be commented out, or have a disclaimer added.

Reply:
See #69 ; This will make PRs conflicting (as would any other proposal to fix this).

The rest of the changes are fine, but recommending end-users to sign up with GitHub is a horrible idea:

- It's confusing as they won't know what GitHub is.
- It's bad because it encourages them to check out GitHub which will lead to them treating it like a forum (bogus bug reports / help requests).
- It's a temporary situation and GitHub should not be required in the first place.
- We don't know how long this work will even still work and it still needs to be manually updated for each XQEMU release.